src/common/catalog/src/consts.rs

```diff
@@ -95,6 +96,13 @@ pub const INFORMATION_SCHEMA_TABLE_CONSTRAINTS_TABLE_ID: u32 = 30;
 pub const INFORMATION_SCHEMA_CLUSTER_INFO_TABLE_ID: u32 = 31;
 /// ----- End of information_schema tables -----
 
+/// ----- Begin of pg_catalog tables -----
+/// mask for pg_catalog tables id, to reserve enough table_id for tables, 0x200 == 256 == 0b100000000
+pub const PG_CATALOG_TABLE_ID_MASK: u32 = 0x200;
+/// id for pg_catalog.pg_class
+pub const PG_CATALOG_PG_CLASS_TABLE_ID: u32 = PG_CATALOG_TABLE_ID_MASK | 1;
+pub const PG_CATALOG_PG_TYPE_TABLE_ID: u32 = PG_CATALOG_TABLE_ID_MASK | 2;
+/// ----- End of pg_catalog tables -----
```

Comment:
@killme2008 @waynexia do we use this MASK falvor? Or simply reserved a range of table id, or even just mixed with other preserved table id?

I'd prefer to simply mixed with other preserved table id.

Reply:
Mixing them looks ok to me. Information schema is also mixed with some old system tables.